script/user-group

```diff
@@ -94,6 +102,9 @@ else
   CMD=""
   if test "$IS_FREEBSD" -eq 1; then
     CMD="pw useradd $USERNAME -c $USERNAME"
+  elif test $IS_DARWIN -eq 1; then
+    USERID=601
+    CMD="sh $SRCDIR/script/darwin-create-user $USERNAME $USERID $GROUPID"
```

Comment:
it feels a bit inconsistent to call dscl directly in the script for the group creation, but then to have a separate script calling dscl for the user creation. 
Either put all the code inline in the user-group script (no separate script), or have the separate script do everything (e.g. no dscl call in user-group at all).